osc-export/pom.xml

```diff
@@ -339,40 +339,6 @@
 				</plugins>
 			</build>
 		</profile>
-
-		<profile>
```

Comment:
this change should be reverted for the ovf to still function.

It might be working because we have a cached jre-8u131-linux-x64.tar.gz on jenkins. please recheck